packages/message-box-react/README.md

```diff
@@ -7,3 +7,45 @@
 ## Kom i gang
 
 [Lær hvordan du kan ta i bruk Jøkul](https://fremtind.github.io/jokul/developer/getting-started/)
+
+## Bruk av React-pakken
+
+### Installasjon
+
+1. Installér pakken med `yarn add @fremtind/jkl-message-box-react` eller `npm i @fremtind/jkl-message-box-react`. Stil-pakken blir automatisk installert som en avhengighet.
+
+2. Importer _både_ React-pakken (velg komponenten(e) du trenger) og stil-pakken i prosjektet ditt:
+
+```js
+import { InfoMessage, ErrorMessage, WarningMessage, SuccessMessage } from "@fremtind/jkl-message-box-react";
+import "@fremtind/jkl-message-box/message-box.min.css";
+```
+
+### Bruk
+
+Komponentene tar følgende props:
+
+-   `title`: **Påkrevd**. Overskriften til meldingsboksen. `string`
+-   `children`: **Påkrevd**. Innholdet i meldingsboksen. `ReactNode`
+-   `fullWidth`: Angir om meldingsboksen skal ta opp hele bredden av beholderen den er inne i. `boolean`
+-   `className`: Eventuell(e) css-klassenavn for komponenten. `string`
```

Comment:
Jeg er litt redd for at dette kommer til å havne i usync med koden :/

Reply:
Ser som sagt bekymringen, men tror fordelene kan oppveie foreløpig.. Vanskelig å linte dette, men en manuell sjekk ved PR burde jo kunne ta det meste